src/main/io/gps.h

```diff
@@ -67,23 +67,11 @@ typedef enum {
     CFG_MOT_GNSSSPEED_THRS = 0x20250038, // U1
     CFG_MOT_GNSSDIST_THRS = 0x3025003b, // U2
     CFG_ANA_USE_ANA = 0x10230001, // L
-    CFG_MSGOUT_NMEA_ID_VTG_I2C = 0x209100b0, // U1
-    CFG_MSGOUT_NMEA_ID_VTG_SPI = 0x209100b4, // U1
     CFG_MSGOUT_NMEA_ID_VTG_UART1 = 0x209100b1, // U1
-    CFG_MSGOUT_NMEA_ID_GSV_I2C = 0x209100c4, // U1
-    CFG_MSGOUT_NMEA_ID_GSV_SPI = 0x209100c8, // U1
     CFG_MSGOUT_NMEA_ID_GSV_UART1 = 0x209100c5, // U1
-    CFG_MSGOUT_NMEA_ID_GLL_I2C = 0x209100c9, // U1
-    CFG_MSGOUT_NMEA_ID_GLL_SPI = 0x209100cd, // U1
     CFG_MSGOUT_NMEA_ID_GLL_UART1 = 0x209100ca, // U1
-    CFG_MSGOUT_NMEA_ID_GGA_I2C = 0x209100ba, // U1
-    CFG_MSGOUT_NMEA_ID_GGA_SPI = 0x209100be, // U1
     CFG_MSGOUT_NMEA_ID_GGA_UART1 = 0x209100bb, // U1
-    CFG_MSGOUT_NMEA_ID_GSA_I2C = 0x209100bf, // U1
-    CFG_MSGOUT_NMEA_ID_GSA_SPI = 0x209100c3, // U1
     CFG_MSGOUT_NMEA_ID_GSA_UART1 = 0x209100c0, // U1
-    CFG_MSGOUT_NMEA_ID_RMC_I2C = 0x209100ab, // U1
-    CFG_MSGOUT_NMEA_ID_RMC_SPI = 0x209100af, // U1
```

Comment:
These are ublox commands that allow us to enable or disable NMEA.  Some GPS Modules have NMEA commands included.  We should retain these commands.

Reply:
Only commands with UART was used - not I2C / SPI. Will restore them if preferred.

Reply:
I'm not fussy, I guess people could find them in old PRs, so they could be removed from here.  @freasy what do you think?